config/setting_gnina_interaction.yaml

```diff
@@ -0,0 +1,78 @@
+# Basic setting
+c_val: 1.0
+# threshold_type: [time, generation_num]
+threshold_type: generation_num
+#hours: 0.01
+generation_num: 300
+output_dir: result/example_gnina_interaction01
+model_setting:
+  model_json: model/model_chembl220k.tf25.json
+  model_weight: model/model_chembl220k_rand.tf25.best.ckpt.h5
+token: model/tokens_chembl220k_rand.pkl
+reward_setting:
+  reward_module: reward.gnina_interaction_reward
+  reward_class: Gnina_interaction_reward
+
+# Advanced setting
+expansion_threshold: 0.995
+simulation_num: 3
+flush_threshold: -1
+policy_setting: 
+  policy_module: policy.ucb1
+  policy_class: Ucb1
+
+# Restart setting
+save_checkpoint: False
+restart: False
+checkpoint_file: chemtsv2.ckpt.pkl
+
+# Filter setting
+use_lipinski_filter: False
+lipinski_filter:
+  module: filter.lipinski_filter
+  class: LipinskiFilter
+  type: rule_of_5
+use_radical_filter: True
+radical_filter:
+  module: filter.radical_filter
+  class: RadicalFilter
+use_pubchem_filter: True
+pubchem_filter:
+  module: filter.pubchem_filter
+  class: PubchemFilter
+use_sascore_filter: False
+sascore_filter:
+  module: filter.sascore_filter
+  class: SascoreFilter
+  threshold: 3.5
+use_ring_size_filter: False
+ring_size_filter:
+  module: filter.ring_size_filter
+  class: RingSizeFilter
+  threshold: 6
+use_pains_filter: False
+pains_filter:
+  module: filter.pains_filter
+  class: PainsFilter
+  type: [pains_a]
+include_filter_result_in_reward: False
+
+# User setting
+# Setting for GNINA
+gnina_bin_path: "reward/gnina.sif"
+gnina_receptor: /scr/5dxh_prep_rec2.pdb
+gnina_autobox_ligand: /scr/5h2_lig.mol2
+gnina_cpus: 4
+gnina_num_modes: 9
+gnina_base_smina_affinity: -10.01144
+gnina_base_cnn_affinity: 8.1709308624
+
+# Setting for prolif
```

Comment:
Modify the key names related to ProLIF as follows to clarify that they are ProLIF options.

prolif_interactions: ...
prolif_tolerance: ...

Reply:
Rename keys in e129ac4adeb1b5a670eac7fac4482c33fc761a38.